src/gettext_boost.cpp

```diff
@@ -30,7 +33,7 @@
 #pragma GCC diagnostic ignored "-Wmissing-field-initializers"
 #elif defined(_MSC_VER)
 #pragma warning(push)
-#pragma warning(disable: 4714)
```

Comment:
4714 was originally down here
4459 was needed so I added it
then I noticed 4714 was not working here, so I moved it